docs/section11/Rule11-15.md

```diff
@@ -0,0 +1,70 @@
+(2) SWH energy consumption can be demonstrated to be reduced by reducing the required temperature of service mixed water, by increasing the temperature, or by increasing the temperature of the entering makeup water. "  
+
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, (g)  
+
+**Evaluation Context:** Each SWH use  
+**Data Lookup:**   
+**Function Call:**
+- **get_SWH_uses_associated_with_each_building_segment**  
+- **get_component_by_id**  
+
+**Applicability Checks:**
+- create a list of swh use ids that need to be checked - we'll create the list from both the proposed and baseline models: `swh_use_ids = []`
+- look at each building in the B_RMD: `for building in B_RMD:`
+    - look at each building segment in the building: `for building_segment in building.building_segments:`
+    - get the service water heating uses in the building segment `service_water_heating_use_ids = get_SWH_uses_associated_with_each_building_segment(P_RMD, building_segment.id)`
+    - look at each service water heating use id: `for swh_use_id in service_water_heating_use_ids:`
+        - append the swh_use_id to the list of swh use ids: `swh_use_ids.append(swh_use_id)`
+- look at each building in the P_RMD: `for building in P_RMD:`
+    - look at each building segment in the building: `for building_segment in building.building_segments:`
+    - get the service water heating uses in the building segment `service_water_heating_use_ids = get_SWH_uses_associated_with_each_building_segment(P_RMD, building_segment.id)`
+    - look at each service water heating use id: `for swh_use_id in service_water_heating_use_ids:`
+        - append the swh_use_id to the list of swh use ids if it's not already in the list: `if !swh_use_id.in sw_use_ids: swh_use_ids.append(swh_use_id)`
+
+- look at each swh_use_id in the swh_use_ids list.  Each of these is applicable (not sure how this works, when an ID is applicable instead of an object): `for swh_use_id in swh_use_ids: CONTINUE TO RULE LOGIC`
+- if there are no swh_use_ids in the list, the rule is not applicable: `if len(swh_use_ids) == 0: NOT_APPLICABLE`
+
+    ## Rule Logic:
+    - get the proposed swh use: `swh_use_p = get_component_by_id(swh_use_id, P_RMD)`
+    - get the baseline swh use: `swh_use_b = get_component_by_id(swh_use_id, B_RMD)`
+    - get the proposed distribution system that serves the swh use: `swh_dist_sys_p = swh_use_p.served_by_distribution_system`
+    - get the baseline distribution system that serves the swh use: `swh_dist_sys_b = swh_use_b.served_by_distribution_system`
+    - create a value to track the rule status: `rule_status = "PASS"`
+    - create a variable to track any notes: `rule_note = ""`
+    - if swh_use_b is null: `if swh_use_b == nil:`
+        - set the rule status to FAIL and provide note: `rule_status = "FAIL"; note = swh_use_id + "exists in the Proposed model, but not in the baseline.`
```

Comment:
seems like if the notes are collecting a list of potentially multiple failures then we should include the use id in each note.

Reply:
the rule is currently evaluate for each swh_use_id.  Not sure how it works for Weili's team, but the reason I flagged it here and not for the others is that in this case, there is a mismatch between the two RMDs

Reply:
I see! Thanks for clarifying